userguide/tutorials/payment_plugin.adoc

```diff
@@ -79,34 +72,33 @@ The following table lists the methods in https://github.com/killbill/killbill-pl
 
 A few pointers for implementing these methods:
 
-* Almost all the `PaymentPluginApi` methods accept parameters like *Kill Bill account Id*, *Kill Bill payment id*, *Kill Bill transaction id*, etc. which are self-explanatory. These may or may not be used by your plugin code. But you may need some of these values to populate the <<PaymentTransactionInfoPlugin, PaymentTransactionInfoPlugin>> object which is returned by most of the methods.
+* Almost all the methods accept parameters like `kbAccountId`, `kbPaymentId`, `kbTransactionId`, etc. which are self-explanatory. These may or may not be used by your plugin code. But you may need some of these values to populate the `PaymentTransactionInfoPlugin` object which is returned by most of the methods (Refer to the <<PaymentTransactionInfoPlugin, "PaymentTransactionInfoPlugin">> section).
 
-* All the methods listed in the `PaymentPluginApi` also accept `Iterable<PluginProperty> properties` as a parameter. A https://github.com/killbill/killbill-api/blob/4ae1c343a593de937415e21feecb9f5405037fa3/src/main/java/org/killbill/billing/payment/api/PluginProperty.java[PluginProperty] consists of a *key-value* pair. It can be used to pass plugin specific properties to the plugin. For example, if your plugin requires properties like *city=San Francisco* and *billing_address=abc*, a client application can create two `PluginProperty` objects and pass a `List` of these objects.
+* All the methods accept `Iterable<PluginProperty> properties` as a parameter. These can be used to pass arbitrary data to the plugin method as explained in the <<plugin_property_notes, "Plugin Property Notes">> section. 
```

Comment:
```suggestion
* All the methods accept `Iterable<PluginProperty> properties` as a parameter. These can be used to pass arbitrary data to the plugin method as explained in the <<plugin_property_notes, Plugin Property Notes>> section. 
```

Reply:
No worries Mary! I had formatted the doc based on the old style guide recommendations, I should have checked the latest guide.